pkg/types/float_slice.go

```diff
@@ -12,6 +12,10 @@ func (s *Float64Slice) Push(v float64) {
 	*s = append(*s, v)
 }
 
+func (s *Float64Slice) Update(v float64) {
+	*s = append(*s, v)
```

Comment:
oh this Update method is already duplicated (Push does the same thing)

Reply:
Push remains unchanged for backward compatibility